### PR TITLE
[api][m]: Calling publisher API to get publisher information and data…

### PR DESCRIPTION
**Pull request by subhankarb**
… package list - refs #58

- removed mock data
- removed mock api and put axiox get call to get data from backend

### CODE REVIEW

File: src/api/publisherPageApi.js [hunk 1]
```diff
@@ -1,47 +1,12 @@
...
-
 class PublisherPageApi {
   static getAllPackages(publisher) {
-    return new Promise((resolve, reject) => {
-      setTimeout(() => {
-        resolve(Object.assign([], packages));
-      }, 100);
-    });
-    // return axios.get('/api/package/'+publisher);
+    return axios.get(`https://staging.datapackaged.com/api/search/package?q=* publisher:${publisher}`);
```
Comment (Fak3) [inline src/api/publisherPageApi.js:5]: We have to move the base url to the central place to allow configuration.

File: src/components/common/PackageList.js [hunk 2]
```diff
@@ -3,6 +3,7 @@ import PackageListItem from "./PackageListItem";
 import {Col, Row, Button} from "react-bootstrap";
 
 const PackageList = ({publisherName, packageList}) => {
+  debugger;
```
Comment (Fak3) [inline src/components/common/PackageList.js]: forgot to remove

**Review by Fak3 — CHANGES_REQUESTED** [review]

File: src/components/common/PackageList.js (hunk 2 above)
Comment (subhankarb) [reply src/components/common/PackageList.js]: removed it

File: src/api/publisherPageApi.js (hunk 1 above)
Comment (subhankarb) [reply src/api/publisherPageApi.js:5]: @Fak3 I am looking how to load the external configuration in webpack. I already raised an issue for that https://github.com/frictionlessdata/dpr-js/issues/84

**Pull request merged**